config/breadcrumbs.rb

```diff
@@ -23,6 +23,11 @@
   link t("space.agreements.new.link_to")
 end
 
+crumb :edit_space_agreement do |agreement|
+  parent :edit_space, agreement.space
+  link t("space.agreements.edit.link_to")
```

Comment:
```suggestion
  link t("space.agreements.edit.link_to", name: agreement.name)
```

Reply:
You should probably accept this suggestion.

Reply:
I should, I just got distracted by Paying Work 🤣